zerver/views/user_settings.py

```diff
@@ -87,6 +83,26 @@ def json_change_settings(request, user_profile,
 
 @authenticated_json_post_view
 @has_request_variables
+def json_change_settings(request, user_profile,
+                         full_name=REQ(default=""),
+                         old_password=REQ(default=""),
+                         new_password=REQ(default=""),
+                         confirm_password=REQ(default="")):
+    # type: (HttpRequest, UserProfile, text_type, text_type, text_type, text_type) -> HttpResponse
+    return change_user_settings(request, user_profile, full_name, old_password,
+                                new_password, confirm_password)
+
+@has_request_variables
+def json_change_user_full_name(request, user_profile, full_name=REQ(default=""),
+                               email=REQ(default="")):
+    # type: (HttpRequest, UserProfile, text_type, text_type) -> HttpResponse
+    target_user = get_user_profile_by_email(email)
+    result = change_user_settings(request, target_user, full_name)
+    do_update_full_name_in_admin_ui(user_profile, target_user, full_name)
+    return result
```

Comment:
This code will let anyone update another user's name!  You should add the `@require_realm_admin` decorator to this method.

Also, you should add error handling to throw an error if the target user doesn't exist or is not part of the realm that the calling user is an administrator for (and we should add tests for this too!).